client/galaxy/scripts/apps/login.js

```diff
@@ -0,0 +1,39 @@
+
+var jQuery = require( 'jquery' ),
+    $ = jQuery,
+    PANEL = require( 'layout/panel' ),
+    _l = require( 'utils/localization' ),
+    PAGE = require( 'layout/page' );
+
+window.app = function app( options, bootstrapped ){
```

Comment:
I'm out right now, but will have it PR'd asap -- just wanted to drop that info.  Meanwhile, there are some good basic examples of bootstrap-tour using multipage on the homepage I think, if that's what you want to see